requirements/development.txt

```diff
@@ -4,4 +4,24 @@
 #
 #    make upgrade
 #
-modernize==0.7
```

Comment:
Previously, the Makefile was missing `pip-compile --upgrade -o requirements/development.txt requirements/development.in`, hence all the new requirements here.